values.yaml

```diff
@@ -0,0 +1,38 @@
+redis:
+  enabled: true
+  auth:
+    enabled: false
+  architecture: standalone
+  master:
+    persistence:
+      enabled: true
+      size: 1Gi  # Adjust the size as needed
+
+onetimesecret:
+  image: 
+    registry: docker.io
+    repository: onetimesecret/onetimesecret
+    tag: latest #will be overwritten by Chart.yaml appVersion
+  service:
+    type: LoadBalancer
+  env:
+    COLONEL: "admin@example.com"
+    SSL: "false"  # Ensure this is a string
+
+### REDIS_URL and HOST Paramerters are dynamically getting updated from templates/onetimesecret-deployment.yaml
+### Expamples for setting SMTP PARAMETERS
+#   SMTP_PORT: "587"
+#   SMTP_USERNAME: "username"
+#   SMTP_PASSWORD: "password"
+#   FROM_EMAIL: "admin@example.com"
+#   TO_EMAIL: "recipent@example.com"
+
+
+ingress:
+  enabled: true
+  ingressClassName: "ontimesecret-ingress"
```

Comment:
Hi, I think you should leave this blank by default, so it uses the default cluster ingress class. And if needed, people can override it.

Reply:
@fad3t if you have one handy, can you share an example helm chart that does this already?

Reply:
you are right, I removed it from here for default values and if user needed it they can now override.